### ISSUE TITLE
Partitioning of the validation set

### ISSUE THREAD

**Issue by liuhhx** [rooneysh/Labelme2YOLO#5]
I found that the training set and the validation set have intersection.
